core/server/data/schema/fixtures/fixtures.json

```diff
@@ -486,6 +486,11 @@
                     "name": "Delete snippets",
                     "action_type": "destroy",
                     "object_type": "snippet"
+                },
+                {
+                    "name": "Reset all passwords",
+                    "action_type": "resetAllPasswords",
```

Comment:
I'd give the permission naming a hard think here. What alternative names have been considered here?

Reply:
Here is the full list of what's used as an `action_type` value: `"activate", "add", "assign", "auth", "backupContent", "browse", "deleteAllContent", "destroy", "download", "edit", "exportContent", "generate", "importContent", "publish", "read", "retry", "send", "sendTestEmail", "upload"`. With this context I feel it's ok to name the action that resets all passwords `resetAllPasswords` 😅 
Do you have an example of what would be more fitting?

Reply:
Yup! Now it totally makes sense, specially that we have `deleteAllContent` :+1: